config.yaml

```diff
@@ -42,8 +42,12 @@ s3:
   acl: public-read
 
 registry_repos:
-  oscontainer: quay.io/fedora/fedora-coreos
-  kubevirt: quay.io/fedora/fedora-coreos-kubevirt
+  oscontainer:
+	registry: quay.io/fedora/fedora-coreos
```

Comment:
Hmm, I guess technically this key should be called `repo` rather than `registry`?